resources/views/forum/topics/_post_info.blade.php

```diff
@@ -65,6 +65,11 @@ class="forum__user-flag forum__user-flag--country"
         </div>
 
         <div class="forum-post__info-panel-extra-bottom">
+            <div class="forum-post__info-row">
+                <a class="forum-post__info-link" href="{{ route("users.posts", $user) }}">
+                    {{ trans_choice('forum.post.info.post_count', $user->user_posts) }}
```

Comment:
As it turned out, this errors out on reply since the value is raw sql statement and can't be converted to integer.

Reply:
should I change `db_unsigned_increment` so that it handles all of the database logic too? I'm not sure how to fix this